onnx_tf/backend_tf_module.py

```diff
@@ -124,24 +115,8 @@ def __init__(self, node, backend):
     self.node = node
     self.backend = backend
     self.handlers = backend._get_handlers(opset=None)
-    self.handler_variables = self._create_handlers_variables(dict())
-
-  def _create_handlers_variables(self, vars_dict):
-    if self.handlers:
-      handler = self.handlers[self.node.domain].get(
-          self.node.op_type,
-          None) if self.node.domain in self.handlers else None
-      if handler and bool(
-          handler.get_req_vars_template(self.node, self.node.attrs)):
-        for v_name, v_template in handler.get_req_vars_template(
-            self.node, self.node.attrs).items():
-          v_init, v_shape = v_template
-          v_name = get_variable_name(self.node, v_name)
-          vars_dict[v_name] = tf.Variable(v_init,
-                                          dtype=v_init.dtype,
-                                          shape=v_shape,
-                                          name=v_name)
-    return vars_dict
+    self.handler_variables = TFModuleHelper._create_handler_variables_for_node(
+        self.handlers, node, init_dict=dict(), var_dict=dict())
```

Comment:
Providing dict() for init_dict and var_dict seems redundant. These are optional parameters with defaults in the helper function. Please consider remove it.